ci-operator/config/openshift/release/openshift-release-master__ci-4.9-upgrade-from-stable-4.8.yaml

```diff
@@ -45,9 +45,12 @@ tests:
     cluster_profile: azure4
     workflow: openshift-upgrade-azure
 - as: e2e-ovirt-upgrade
-  interval: 48h
+  interval: 6h
   steps:
     cluster_profile: ovirt
+    leases:
```

Comment:
I wonder if we can avoid setting leases in the test. Can it be automatically set in workflow/cluster_profile?

Reply:
Thanks for the tip, I didn't know we can define them on the workflow level :) fixed